### ISSUE TITLE
Normilize the dataset for yolo training

### ISSUE THREAD

**Issue by ghost** [ultralytics/yolov5#4873]
## ❔Question
i have the image and i have label the image using opencv 
x,y,w,h = cv2.boundingRect(cntr)
 cv2.rectangle(result, (x, y), (x+w, y+h), (0, 0, 255), 2)
i have save the coordinates in txt file, like     0 164 487 105 35
 this 
 How can i normilze it and train use it for training? Is it work or not?
## Additional contex
![1](https://user-images.githubusercontent.com/70446516/134164532-64d95fcf-85b2-4a63-acd8-d0699db28595.jpg)

Here is the text file

[1.txt](https://github.com/ultralytics/yolov5/files/7202741/1.txt)

**Comment by glenn-jocher** [comment]
@hammadyounas2008 see Train Custom Data tutorial for instructions on making normalized labels:

### 2. Create Labels

After using a tool like [CVAT](https://github.com/opencv/cvat) or [makesense.ai](https://www.makesense.ai/) to label your images, export your labels to **YOLO format**, with one `*.txt` file per image (if no objects in image, no `*.txt` file is required). The `*.txt` file specifications are:

- One row per object
- Each row is `class x_center y_center width height` format.
- Box coordinates must be in **normalized xywh** format (from 0 - 1). If your boxes are in pixels, divide `x_center` and `width` by image width, and `y_center` and `height` by image height.
- Class numbers are zero-indexed (start from 0).

<img width="800" alt="Image Labels" src="https://user-images.githubusercontent.com/26833433/91506361-c7965000-e886-11ea-8291-c72b98c25eec.jpg">

The label file corresponding to the above image contains 2 persons (class `0`) and a tie (class `27`):

<p align="center"><img width="428" src="https://user-images.githubusercontent.com/26833433/112467037-d2568c00-8d66-11eb-8796-55402ac0d62f.png"></p>


## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)

**Comment by ghost** [comment]
The above file does not work for training?

**Comment by ghost** [comment]
@glenn-jocher  Will it works for training or I have to do that again? i don't want manual anootation

**Comment by glenn-jocher** [comment]
@hammadyounas2008 I don’t know what you are asking. Directions for labeling are already cited above.

**Comment by ghost** [comment]
I am asking did this file will work for training?
[1.txt](https://github.com/ultralytics/yolov5/files/7209858/1.txt)

**Comment by glenn-jocher** [comment]
@hammadyounas2008 if it follows the guidelines it will work.

**Comment by ghost** [comment]
Okay.

**Comment by hammad2008** [comment]
@glenn-jocher  How to print FPS on video?

**Comment by glenn-jocher** [comment]
@hammad2008 1.0 / inference_time.
